charms/openfga-k8s/tests/integration/conftest.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+import pytest_asyncio
+from pytest_operator.plugin import OpsTest
+from utils import fetch_charm
+
+logger = logging.getLogger(__name__)
+
+
+@pytest_asyncio.fixture(scope="module")
+async def charm(ops_test: OpsTest):
+    logger.info("Building local charm")
+    charm = await fetch_charm(ops_test, "*.charm", ".")
```

Comment:
Not in this case, the name of the charm will change depending on the series it supports, etc. And in this case the charms are in different folders so it all works just fine. There's also a check so that if the folder has multiple .charm files, we fail early.